From 611a0b3b2047bf8a49db32dc4b30684a10f5b6eb Mon Sep 17 00:00:00 2001
From: Mike Taves <mwtoews@gmail.com>
Date: Mon, 14 Dec 2020 12:06:36 +1300
Subject: [PATCH] Expand CI and tests to support GEOS 3.9.0beta2 (#1042)

* Expand CI and tests to support GEOS 3.9.0beta2

* Change build/install scripts to be CI agnostic

* disable AppVeyor's ctest fail due to GEOS issue with simplify on MSVC

* find_library() for Linux should use 'libgeos_c'
---
 tests/test_svg.py | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/tests/test_svg.py b/tests/test_svg.py
index fcc2679..dac3e71 100644
--- a/tests/test_svg.py
+++ b/tests/test_svg.py
@@ -174,7 +174,8 @@ class SvgTestCase(unittest.TestCase):
         self.assertSVG(GeometryCollection(), '<g />')
         # Valid
         self.assertSVG(
-            Point(7, 3).union(LineString([(4, 2), (8, 4)])),
+            GeometryCollection(
+                [Point(7, 3), LineString([(4, 2), (8, 4)])]),
             '<g><circle cx="7.0" cy="3.0" r="3.0" stroke="#555555" '
             'stroke-width="1.0" fill="#66cc99" opacity="0.6" />'
             '<polyline fill="none" stroke="#66cc99" stroke-width="2.0" '
